netbox/tenancy/tables/columns.py

```diff
@@ -1,9 +1,12 @@
 import django_tables2 as tables
 
+from netbox.tables import columns
+
 __all__ = (
     'TenantColumn',
     'TenantGroupColumn',
     'TenancyColumnsMixin',
+    'ContactsColumnMixin',
```

Comment:
Try to keep these in order alphabetically, just to help readability.

```suggestion
    'ContactsColumnMixin',
    'TenantColumn',
    'TenantGroupColumn',
    'TenancyColumnsMixin',
```